src/Typescript/Server.ts

```diff
@@ -28,30 +28,34 @@ interface TurnInfo {
 
 export function turnTaken(turnInfo: TurnInfo) {
     var turn = document.createElement("div");
-    turn.textContent = "Player " + players[currentPlayer].position + " "
+    turn.className += "mb-2";
+    turn.textContent = players[currentPlayer].position + " "
             + (turnInfo.clueType
-                    ? "Clued player " + turnInfo.player.position + " on "
-                    : turnInfo.discard ? "Discarded" : "Played") + " ";
+                    ? "clued " + turnInfo.player.position + " on "
+                    : turnInfo.discard ? "discarded" : "played") + " ";
     if (turnInfo.tile) {
         var clueOnNumber = typeof(turnInfo.clueType) === "number";
         var clueOnColor = typeof(turnInfo.clueType) === "string";
-        var textContent = String(turnInfo.tile.number) + (turnInfo.clueType ? "s" : "");
+        var textContent = String(turnInfo.tile.number) + (turnInfo.clueType ? "" : "");
```

Comment:
since the ternary operator (the `? ... : ...` thing) returns `""` for both, you can just get rid of the `+ (turnInfo.clueType ... )` bit altogether